playlist_souffle/entities/souffler.py

```diff
@@ -0,0 +1,29 @@
+"""Module for souffler entity."""
+
+from playlist_souffle.definitions.playlist import Playlist
+from playlist_souffle.entities.souffler_util import (
+    generate_souffle_description,
+    generate_souffle_name,
+    souffle_tracks
+)
+
+def generate_souffled_playlist(original_playlist, user_id, shuffle_by, collection_tracks_by_track, souffle_time):
```

Comment:
why not `souffle_playlist`?

Reply:
I thought you recommended this name in a previous comment

Reply:
that's very possible.

Reply:
In my mind (and may not be the best instict) but souffle_playlist (since its the same name as the lambda and use case function) implies creating the souffled playlist and creating it on the spotify service.

This function just generates the playlist but doesnt create it on spotify

Reply:
👍